DailyReportWeb/Web.config

```diff
@@ -98,7 +98,7 @@
   </runtime>
   <system.webServer>
     <modules runAllManagedModulesForAllRequests="true">
-      <remove name="WebDAVModule"/>
+      <remove name="WebDAVModule" />
```

Comment:
In comit-ul lui Sebi era doar un spatiu adaugat. 
Da, tin minte ca mi-ai zis. 
E super.